ui/redux/selectors/content.js

```diff
@@ -123,7 +123,21 @@ export const makeSelectNextUnplayedRecommended = (uri: string) =>
             continue;
           }
 
-          if (!history.some((item) => item.uri === recommendedForUri[i])) {
+          const recommendedUriInfo = parseURI(recommendedUri);
+
+          if (claimsByUri[uri] && claimsByUri[uri].claim_id === recommendedUriInfo.claimId) {
+            // Skip myself (same claim ID)
```

Comment:
I can potentially see someone complaining and thinking their videos are never recommended because they never their own videos autoplay. While it's likely you don't want to autoplay watch your own videos, I think a user would rather notice their video being suggested over thinking it's never suggested.

Reply:
`Unrelated?`

This is not checking against "claims that I own", but solving the "video A recommended for video A" problem.
>_There's also a weird case where the URL differs by just a little (p.09 vs p-09), but with the same claim ID:
lbry://vacuum-tube-computer-p.09-–-building#5212bc8bc63c373e2bf1ebc5b765595ed7b6514d
lbry://vacuum-tube-computer-p-09-–-building#5212bc8bc63c373e2bf1ebc5b765595ed7b6514d_